website/docs/docs/build/packages.md

```diff
@@ -157,9 +157,60 @@ packages:
 
 Where `name: 'dbt_utils'` specifies the subfolder of `dbt_packages` that's created for the package source code to be installed within.
 
-### Private packages
+## Private packages
 
-#### SSH Key Method (Command Line only)
+### Native private packages <Lifecycle status='beta'/> 
+
+dbt Cloud supports private packages from [supported](#prerequisites) Git repos leveraging an exisiting [configuration](/docs/cloud/git/git-configuration-in-dbt-cloud) in your environment. Previously, you had to configure a [token](#git-token-method) to retrieve packages from your private repos.  
+
+#### Prerequisites
+
+Native support for private packages is supported on the following services:
+- [Github](/docs/cloud/git/connect-github)
+- [Azure DevOps](/docs/cloud/git/connect-azure-devops)
+
+Support for Gitlab is coming soon. 
+
+To use this feature, you must have a supported Git provider configured in the **Integrations** section of your **Account settings**.
+
+#### Configuration
+
+Use the `private` block in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
```

Comment:
maybe the sme will have more insight but i'd suggest using key instead of block. "private block" might confuse readers because "block" typically refers to multi-line (at least for me it did)
```suggestion
Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
```

Reply:
We use `blocks` elsewhere in the docs for this sort of thing, but it might be a good convo to have in a meeting